minecraft/protocol/chunk_pos.go

```diff
@@ -0,0 +1,34 @@
+package protocol
+
+// ChunkPos is the position of a chunk. It is composed of two integers, and is typically written as either
+// two varint32s.
```

Comment:
Was this copied from somewhere that let the position be written in two ways? You should probably remove the "either"